### PR TITLE
Add production properties

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: environment-properties/prod_properties.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "redirectUris": [
+      "https://tdr.nationalarchives.gov.uk/*"
+    ]
+  },
+  "tdr-user-admin": {
+    "redirectUris": [
+      "https://tdr.nationalarchives.gov.uk/*"
+    ]
+  }
+}
```
Comment (TomJKing) [inline environment-properties/prod_properties.json:36]: Missing line break

**Review by TomJKing — COMMENTED** [review]
Looks good. Just the usual comment :)

File: environment-properties/prod_properties.json (hunk 1 above)
Comment (MancunianSam) [reply environment-properties/prod_properties.json:36]: I've added one in.

Comment (TomJKing) [reply environment-properties/prod_properties.json:36]: can't see one?

Comment (MancunianSam) [reply environment-properties/prod_properties.json]: Sorry, forgot to push

**Review by TomJKing — APPROVED** [review]

**Pull request merged**